src/app/cli/src/full_test.ml

```diff
@@ -106,6 +117,10 @@ let run_test () : unit Deferred.t =
         `With_public_key
           (Public_key.compress largest_account_keypair.public_key)
       in
+      let fee = Currency.Fee.of_int in
+      let snark_work_fee, transaction_fee =
+        if with_snark then (fee 0, fee 0) else (fee 1, fee 2)
```

Comment:
Full-test with snark runs with transaction-capacity = 2 and so, it can't accommodate fee-transfers for transaction-fee or prover-fee. Otherwise, it could be anything